docs/data/trigger-sync.md

```diff
@@ -0,0 +1,270 @@
+
+To set up conditional syncing you need to:
+
+1. Create or use an exisitng module that implements a `sensor` component that uses your custom logic to determine when to enable and disable sync.
+2. Change the configuration of the data manager to enable selective sync.
+
+## Requirements
+
+Before configuring your sensor, you must [create a machine](https://docs.viam.com/fleet/machines/#add-a-new-machine) and you also need to:
+
+1. Enable [data capture](https://docs.viam.com/data/capture/).
+2. Enable [cloud sync](https://docs.viam.com/data/cloud-sync/).
+
+## Add sensor to determine when to sync
+
+In this example, you will configure sync to only trigger during a specific time frame of the day using an existing module [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time).
+If you need to trigger sync based on a different condition, you need to create your own module and adjust the module logic accordingly.
+Additional examples are available in this [GitHub Repo](https://github.com/viam-labs/trigger-sync-examples-v2).
+
+To use [`sync-at-time:timesyncsensor`](https://app.viam.com/module/naomi/sync-at-time):
+
+1. Go to your machine's **Config** page and click **Add Component**.
+2. Then select the `sync-at-time:timesyncsensor` model from the [`sync-at-time` module](https://app.viam.com/module/naomi/sync-at-time).
+3. Click **Add module**, then enter a name for your sensor and click **Create**.
+   The sensor will return true and enable sync when in a specified time frame.
+4. To configure your timeframe, go to the new component panel and copy and paste the following attribute template into your sensor’s **Attributes** box:
+
+   {{< tabs >}}
+   {{% tab name="Config builder" %}}
+
+```json
+{
+  "start": "HH:MM:SS",
+  "end": "HH:MM:SS",
+  "zone": "<TIMEZONE>"
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json
+{
+  "name": "<SENSORNAME>",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "HH:MM:SS",
+    "end": "HH:MM:SS",
+    "zone": "<TIMEZONE>"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json
+{
+  "name": "timesensor",
+  "model": "naomi:sync-at-time:timesyncsensor",
+  "type": "sensor",
+  "namespace": "rdk",
+  "attributes": {
+    "start": "18:29:00",
+    "end": "18:30:00",
+    "zone": "CET"
+  },
+  "depends_on": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for the `naomi:sync-at-time:timesyncsensor` sensor:
+
+<!-- prettier-ignore -->
+| Name    | Type   | Inclusion    | Description |
+| ------- | ------ | ------------ | ----------- |
+| `start` | string | **Required** | The start time for the time frame during which you want to sync, example: `"14:10:00"`.  |
+| `end`   | string | **Required** | The start time for the time frame during which you want to sync, example: `"15:35:00"`. |
```

Comment:
```suggestion
| `end`   | string | **Required** | The end time for the time frame during which you want to sync, for example: `"15:35:00"`. |
```
the "for" is opt but I think without it there should be a period instead of a comma

Reply:
```suggestion
| `end`   | string | **Required** | The end of the sync time frame, for example: `"15:35:00"`. |
```
[opt] Could also shorten both of these

Reply:
Also fix "start" in the readme